### DISCUSSION TITLE
sw average?

### DISCUSSION THREAD

**Discussion by PhD-GOAT** [raphaelvallat/yasa#64] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/93689515/167322605-e0f9a6f3-04fc-4c83-8028-23b7f26c54e6.png)

I want to know what each color means.

**Comment by raphaelvallat** [comment, marked as answer]
Please see the documentation: https://raphaelvallat.com/yasa/build/html/generated/yasa.SWResults.html#yasa.SWResults.plot_average 

```
hue: str
    Grouping variable that will produce lines with different colors. Can be either ‘Channel’ or ‘Stage’.
```

The colors represent different channels
